llama-rs/src/vocabulary.rs

```diff
@@ -65,9 +59,7 @@ impl Vocabulary {
         self.max_token_length = self.max_token_length.max(content.len());
         self.id_to_token.push(content.clone());
         self.id_to_token_score.push(score);
-        if self.token_to_id.insert(content, id).is_some() {
```

Comment:
this is sanity check, maybe don't delete this

Reply:
I added the sanity check - your code wasn't doing it before: https://github.com/rustformers/llama-rs/pull/125/commits/9b908ae57a77f0a134fd2720d66f024f06286ce4#diff-437de44ac54122eab1784619577c65d5d869088c21979e99292d5a089768bceeR68

I removed the check because my GGML Alpaca 7b q4 had several duplicate tokens for ' ' with both loader1 and 2. If models are out there with duplicate terms in the vocabulary, we'll need to support them.

Reply:
interesting